pkg/vm/engine/disttae/logtailreplay/partition_state.go

```diff
@@ -48,7 +48,7 @@ type PartitionState struct {
 	primaryIndex *btree.BTreeG[*PrimaryIndexEntry]
 	//for non-appendable block's memory deletes, used to getting dirty
 	// non-appendable blocks quickly.
-	dirtyRows   *btree.BTreeG[RowEntry]
+	dirtyBlks   *btree.BTreeG[BlockEntry]
```

Comment:
name it dirtyBlocks to keep style persistency.

Reply:
Done